src/math/sincosf.rs

```diff
@@ -128,14 +128,14 @@ pub fn sincosf(x: f32) -> (f32, f32) {
 #[cfg(test)]
 mod tests {
     use super::sincosf;
-    use crate::_eqf;
-
-    #[test]
-    fn with_pi() {
-        let (s, c) = sincosf(core::f32::consts::PI);
-        _eqf(s.abs(), 0.0).unwrap();
-        _eqf(c, -1.0).unwrap();
-    }
+    // use crate::_eqf;
+
+    // #[test]
+    // fn with_pi() {
```

Comment:
Does pi not pass anymore?

Reply:
Because the patched function now produces more accurate results, (`sincosf(core::f32::consts::PI) = (-8.742278e-8, -1)` ), the old test doesn't pass. I'm not really sure what to do with it.

Reply:
Oh, interesting. Maybe just delete the test and leave a `NB` comment for the module, in case we replace this with an implementation that handles it (or just special cases?)